packages/rOpenScPCA/tests/testthat/test-calculate-clusters.R

```diff
@@ -1,9 +1,24 @@
+suppressPackageStartupMessages(library(SingleCellExperiment))
+suppressPackageStartupMessages(library(Seurat))
+
+sce <- scpcaTools:::sim_sce(n_cells = 100, n_genes = 10, n_empty = 0)
 test_mat <- matrix(
   runif(1000, -3, 3),
   nrow = 100,
   ncol = 10
 )
-rownames(test_mat) <- as.character(1:100)
+rownames(test_mat) <- colnames(sce)
+colnames(test_mat) <- paste0("PC_", 1:10) # quiet seurat warnings
+reducedDim(sce, "PCA") <- test_mat
+
+srat <- CreateSeuratObject(counts = counts(sce), assay = "RNA")
+srat[["pca"]] <- CreateDimReducObject(
+  embeddings = test_mat,
+  key = "PC_",
```

Comment:
Is this the default? Or is it what we are creating when we convert?

Reply:
Ah, I think this should have be removed; it's not needed.

Reply:
Ugh nope, it does need it, but it's not relevant to our usage of the Seurat object here. It will forcibly use this to update column names though...